bookCopies/models.py

```diff
@@ -4,7 +4,16 @@
 
 
 # Create your models here.
-class bookCopies(models.Model):
+class BookCopy(models.Model):
     book = models.ForeignKey(Book, on_delete=models.CASCADE)
-    acquisition_date = models.DateField()
+    acquisition_date = models.DateField(
+        help_text='Please select date of acquisition of the book',
```

Comment:
all of such strings should be auto translated when mulitple translation are added in the settings:
e.g in this case:
first import: from django.utils.translation import gettext_lazy as _

acquisition_date = models.DateField(
        help_text=_('Please select date of acquisition of the book'),
        error_messages={
            "empty": _("date is required"),
            "invalid": _("Please enter a valid date"),
        }
    )
  and 
class Meta:
        verbose_name = _("ClassName")


Need to make this change, throughout the project